Task 4/Task4/Task4/CastomGit.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Actions;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Task4
+{
+    class CastomGit
+    {
+        private string path;
+
+        private List<(DateTime, FileAction)> Log = new List <(DateTime, FileAction)>();
+
+        public Dictionary<String, String> FilePastPath = new Dictionary<string, string>();
+
+        private UserCommunication communicator = new UserCommunication();
+
+        private DateTime LogTime;
+
```

Comment:
* public properties after private fields.
* if there's a public field - think about making it a property (to stop others from changing the fields which may lead to unexpected results)